app/Http/Controllers/V2FrontpageController.php

```diff
@@ -12,14 +13,16 @@ public function index()
     {
         $loggedUser = auth()->user();
 
-        $flights = Content::getLatestItems('flight', 9);
+        $flights = Content::getLatestItems('flight', 9, 'id');
```

Comment:
- Why sort on id? Performance?
- Lets split it into separate PR **Frontpage / destination speed improvements**